spacy/cli/train.py

```diff
@@ -328,6 +330,10 @@ def train(
                         gpu_wps=gpu_wps,
                     )
                     msg.row(progress, **row_settings)
+
+            if latest > 0 and i >= latest:
+                shutil.rmtree(output_path / ("model%d" % (i - latest)))
```

Comment:
```suggestion
                shutil.rmtree(output_path / ("model%d" % (i - n_latest)))
```

Also, if I remember correctly, `shutil.rmtree` takes a string, not a path? I think I fixed some issue recently where this caused a warning. We have a `path2str` helper in `spacy.compat` that you can use for this.